### ISSUE TITLE
Check if jumpable

### ISSUE THREAD

**Issue by eichboson** [SirVer/ultisnips#1105]
This is not an issue, but a feature request.
I have tried to somehow get a simple function to return true or false whether a jump is present
or not, but I couldn't get a reliable solution, that worked in every situation. Would it be possible
to implement a function "isJumpable" to do just that?

coc-snippets for example provides coc#jumpable() but I don't want to use coc-snippets with ultisnips as it doesnt support python t s m variables

**Comment by SirVer** [comment]
Yes, this would be feasible. It needs refactoring in https://github.com/SirVer/ultisnips/blob/8ff301c651d6df806c0f9305273a6c62a693bc48/pythonx/UltiSnips/snippet_manager.py#L497 to implement such a functionality.

**Comment by mvcouwen** [comment]
Would this not be the same as checking whether there is an active snippet? For example, by checking `len(self._active_snippets)`?

**Comment by thorlucas** [comment]
This would be a great feature

**Comment by SirVer** [comment]
Closing - this has been done for a while; just not flagged on the issue.

`UltiSnips#CanJumpForwards()` and `UltiSnips#CanJumpBackwards()` (added in #1314) return `1` when a jump is possible and `0` otherwise. They're safe to call from any expression context (including `<expr>` mappings and Nvim Lua) because they don't mutate the buffer. The predicates are documented under [`UltiSnips-trigger-functions`](https://github.com/SirVer/ultisnips/blob/master/doc/UltiSnips.txt) alongside `UltiSnips#CanExpandSnippet()`, and #1661 added regression tests pinning the three so they don't silently regress.

A `coc#jumpable()`-style wrapper can be a one-liner over them:

```vim
function! UltiSnips#Jumpable() abort
  return UltiSnips#CanJumpForwards() || UltiSnips#CanJumpBackwards()
endfunction
```
